tests/load/test_read_interfaces.py

```diff
@@ -801,6 +800,235 @@ def test_ibis_dataset_access(populated_pipeline: Pipeline) -> None:
     ibis_connection.disconnect()
 
 
+@pytest.mark.no_load
+@pytest.mark.essential
+@pytest.mark.parametrize(
+    "populated_pipeline",
+    configs,
+    indirect=True,
+    ids=lambda x: x.name,
+)
+def test_dataset_load_id_retrieval(populated_pipeline: Pipeline) -> None:
+    successful_load_ids = populated_pipeline.list_completed_load_packages()
+    dataset = cast(
+        ReadableDBAPIDataset,
+        _dataset(
+            destination=populated_pipeline.destination,
+            dataset_name=populated_pipeline.dataset_name,
+            # use name otherwise aleph schema is loaded
+            schema=populated_pipeline.default_schema_name,
+            dataset_type="ibis",
+        ),
+    ).items
+    assert isinstance(dataset, ReadableIbisRelation)
+
+    results = dataset.list_load_ids().fetchall()
+    load_ids = [r[0] for r in results]
+    assert set(load_ids) == set(successful_load_ids)
+    # check descending order
+    assert load_ids == sorted(successful_load_ids, reverse=True)
+
+    # check status kwarg
+    assert len(dataset.list_load_ids(status=1).fetchall()) == 0
+    assert len(dataset.list_load_ids(status=[0]).fetchall()) == len(successful_load_ids)
+    assert len(dataset.list_load_ids(status=[0, 1]).fetchall()) == len(successful_load_ids)
+    assert len(dataset.list_load_ids(status=[1, 2]).fetchall()) == 0
+
+    # check limit kwarg
+    assert dataset.list_load_ids(limit=0).fetchall() == []
+    # limit 1 should return the max value (i.e., sort before limit)
+    assert dataset.list_load_ids(limit=1).fetchall()[0][0] == max(successful_load_ids)
+    # 50 is larger than the expected number of `load_id`
+    assert len(dataset.list_load_ids(limit=50).fetchall()) == len(successful_load_ids)
+
+    # check latest_load_id, which is `list_load_ids(limit=1)` and unpacking the list
+    latest_load_id = dataset.latest_load_id().fetchall()[0][0]
+    assert latest_load_id == max(load_ids) == max(successful_load_ids)
+    # TODO we probably want this to return None or empty list?
+    # this would require query post-processing
+    assert dataset.latest_load_id(status=1).fetchall() == [(None,)]
+
+
+@pytest.mark.no_load
+@pytest.mark.essential
+@pytest.mark.parametrize(
+    "populated_pipeline",
+    configs,
+    indirect=True,
+    ids=lambda x: x.name,
+)
+def test_ibis_filter_load_ids(populated_pipeline: Pipeline) -> None:
+    # TODO test filter when >1 successful load_id are present in destination
```

Comment:
I agree with the comment, I add another load run that loads some stuff to the populated table in the setup method, you may need to update tests above